src/functions.js

```diff
@@ -0,0 +1,66 @@
+function capitalizeString(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function reverseString(string) {
+  return string.split('').reverse().join('');
+}
+
+const calculator = (a, b) => {
+  const add = () => a + b;
+  const subtract = () => a - b;
+  const divide = () => a / b;
+  const multiply = () => a * b;
+
+  return {
+    add,
+    subtract,
+    divide,
+    multiply,
+  };
+};
+
+const cipher = (string, shift) => {
+  const alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'];
```

Comment:
What about uppercase letters?

Reply:
As you can review from the Odin project, the uppercase handler is not a requirement so, in this case, it is an optional feature.

Reply:
Oh, sorry about this, my mistake!

Reply:
Oh, sorry about this, my mistake!